defaults/main.yml

```
@@ -32,4 +32,7 @@ geoip2_geoipupdate_service_name: geoipupdate
 # Therefore, the default is to update the databases on Wednesdays and Saturdays
 # between 00:00 and 03:00 EST.
 geoip2_geoipupdate_service_timer_on_calendar: Wed,Sat America/New_York
+# This value must be a valid systemd.time time span value. Please see the
+# documentation for more information:
+# https://www.freedesktop.org/software/systemd/man/latest/systemd.time.html
 geoip2_geoipupdate_service_timer_randomized_delay_sec: 3h
```

See my related comment in `README.md`:
```suggestion
geoip2_geoipupdate_service_timer_randomized_delay: 3h
```

---

I do not want to change this because the name signifies that it is for the `RandomizedDelaySec` option.